packages/react-sdk/src/components/BoardBar/pdfv2/pdf.worker.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { renderPDF } from './pdf.local';
+
+if (import.meta.hot) {
```

Comment:
This is a whole bunch of hacking because HMR for react in vite currently is totally messed up with webworkers.

Reply:
This makes it explicitly error out which is intended here. Otherwise it would crash the webworker when using react.